### PR TITLE
Consistency of task button positions

### PR DESCRIPTION
**Pull request by ronnystandtke**
I noticed that most task dialogs have their buttons north. I also noticed that the angular measure task had its buttons south despite the linear measure task was already changed to have them north. This is now fixed.

I also removed a redundant implementation of the close button in the display properties task (View -> Appearance...). This way it now also has its close button north per default.

The "Edit -> Placement" task still has the buttons at the south position but I am not sure if it is by design because of the additional reset button...

- [X]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8826">pipeline status</a> for feature branch [PR_8826](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8826). Pipeline [802726945](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/802726945) was triggered at [7580878](https://github.com/FreeCAD/FreeCAD/pull/8826/commits/75808785c909acebd8a0283668ce683673f0d266). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by adrianinsaval** [comment]
Thank you, the display properties dialog already has the button on the north side in 0.21, but I think it's also good to remove unnecessary code. I had not noticed the angular dimension discrepancy, which is odd as I had searched the entire code for the "South" keyword before. 

If you have the time, could you also look into moving the buttons for the clipping and persistent section cut dialogs?
image

also the edit->placement dialog:
image

**Comment by donovaly** [comment]
> @ronnystandtke can you provide screenshot(s) of the change you're proposing ?

Yes, this would be good since I cannot see the difference at a first look.

**Comment by luzpaz** [comment]
Does this change tab clicking order (using tab button to navigate the dialog box)?

**Comment by ronnystandtke** [comment]
> @ronnystandtke can you provide screenshot(s) of the change you're proposing ?

OK, here is a screenshot comparing the situation before and after this pull request:
FreeCAD_Angular_Measure_Task_Button_Position_Patch

> Does this change tab clicking order (using tab button to navigate the dialog box)?

I noticed that before the patch it was impossible to move the focus to the close button by pressing the tab button. After applying the patch it was possible.


Regarding the other questions: I will take a look at other parts of FreeCAD in the future, but please don't mix up too many different things in this pull request. I would be glad if we could first merge this PR and continue the discussion in later PRs.

File: src/Gui/DlgDisplayProperties.ui [hunk 1]
```diff
@@ -531,21 +524,5 @@
     </hint>
    </hints>
   </connection>
-  <connection>
-   <sender>buttonBox</sender>
-   <signal>rejected()</signal>
```
Comment (donovaly) [inline src/Gui/DlgDisplayProperties.ui:536]: Why did you remove this signal?

**Comment by donovaly** [comment]
> If you have the time, could you also look into moving the buttons for the clipping and persistent section cut dialogs?

This has nothing to do with the PR as far as I can see. Since I am the author of the section cutting dialog, please CC me in a new forum thread to discuss what could be improved.

**Comment by donovaly** [comment]
> OK, here is a screenshot comparing the situation before and after this pull request: 

Thanks, now I get what is meant by North and South (strange wording since depending on where I look towards, south is up, left, right or down. Thus North/south on a PC screen is misleading). I know it is a technical term you are just use it.

The PR is fine with me except of the removal of the rejected() signal (which I don't understand yet).

File: src/Gui/DlgDisplayProperties.ui (hunk 1 above)
Comment (ronnystandtke) [reply src/Gui/DlgDisplayProperties.ui:536]: I removed the whole buttonBox widget. It is redundant because the same thing is already implemented in DlgDisplayPropertiesImp.cpp with:
```
QDialogButtonBox::StandardButtons TaskDisplayProperties::getStandardButtons() const
{
    return QDialogButtonBox::Close;
}
```

**Comment by donovaly** [comment]
@chennes could you please have a look at this PR? Looks OK for me but maybe I miss something.

**Comment by chennes** [comment]
> Regarding the other questions: I will take a look at other parts of FreeCAD in the future, but please don't mix up too many different things in this pull request. I would be glad if we could first merge this PR and continue the discussion in later PRs.

Agreed! Let's keep these PRs tight and focused (as this one is). Everything looks great to me, thanks for the PR.
